src/base.ts

```diff
@@ -41,76 +30,79 @@ const withPagination = <T>(json: {
   return { content, fieldsStats, pagination, topListing }
 }
 
-export const resolveServiceUrl = (service: Service): string => {
-  let url: string
-  switch (service) {
-    case Service.CAR:
-      url = apiClient.configuration.carServiceUrl
-      break
-    case Service.CATALOGUE:
-      url = apiClient.configuration.catalogueServiceUrl
-      break
-    case Service.SEARCH:
-      url = apiClient.configuration.searchServiceUrl
-      break
-    case Service.DEALER:
-      url = apiClient.configuration.dealerServiceUrl
-      break
-    case Service.OPTION:
-      url = apiClient.configuration.optionServiceUrl
-      break
-    case Service.ANALYTICS:
-      url = apiClient.configuration.analyticsServiceUrl
-      break
-    case Service.USER_NOTIFICATION:
-      url = apiClient.configuration.userNotificationServiceUrl
-      break
-    case Service.TOKEN_REFRESH:
-      url = apiClient.configuration.tokenRefreshServiceUrl
-      break
-    default:
-      throw new Error(`Tried to resolve url of unknown service "${service}"`)
+const authorizationHeader = () => {
+  if (!apiClient.tokens.accessToken) {
+    return null
   }
 
-  if ((url || "").trim()) {
-    return url
-  }
+  return `Bearer ${apiClient.tokens.accessToken}`
+}
 
-  throw new Error(`Missing endpoint configuration for "${service}" service`)
+export interface ApiCallOptions extends Omit<RequestInit, "method" | "body"> {
```

Comment:
What's the reasoning behind naming Request options Call?

Reply:
We never use the word `Request`. In the context of the API client, we always talk about API calls so I liked @masone's suggestion